### DISCUSSION TITLE
JIT value syntax for grid columns (col-span)

### DISCUSSION THREAD

**Discussion by mrtnpro** [tailwindlabs/tailwindcss#4381] [Help]
I can't figure out how to use JIT in combination with col-span-[].
None of the following variants have worked for me:
* `grid-span-[17]`
* `grid-span-[span 17]`
* `grid-span-[span 17/span 17]` 
* `grid-span-[span 17,span 17]`.

None of them seem to generate any classes. Maybe someone can point me in the right direction (if this is possible at all with JIT). Thanks!

**Comment by simonswiss** [comment]
Hey!

I have just [opened a pull request](https://github.com/tailwindlabs/tailwindcss/pull/5148) that will add support for this, once merged.

The utilities you mentioned are generated by the `gridColumn` plugin, and the class itself (the prefix) is only `col-`. The `span-` part is part of the _modifier_ for each value, so the arbitrary value will be a little different:

```grid-[span,17,/,span,17]```

Since arbitrary values [cannot contain spaces](https://tailwindcss.com/docs/just-in-time-mode#arbitrary-value-support), you need to use commas here to obtain the correct syntax.

![image](https://user-images.githubusercontent.com/485747/128117040-444d4c26-2427-405f-9f9a-75ea185dea0d.png)

That syntax won't work just yet, but this PR brings support for it 👍
